src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -37,7 +37,7 @@
 
 			<TransitionGroup name="radial-reveal">
 				<!-- Filters -->
-				<div v-show="!isFocused" key="filters" class="filters">
+				<div v-show="searchText === ''" key="filters" class="filters">
```

Comment:
I wouldn't make it visible depending on `searchText` because the buttons are accessible and visible when you filter. 

https://github.com/nextcloud/spreed/assets/84044328/bfc82bb9-c7f5-41dd-a6a7-28a1b99593f1

Reply:
```vue
					<NcActions class="filter-actions"
						:primary="isFiltered !== null && !isFocused">
```
How about this?

Reply:
but you can still click on it when the search box is expanded. Maybe shrink the button to be hidden by the search box?

Reply:
or return back to `visually-hidden` class